styles/colors-silicon-day.less

```diff
@@ -130,6 +130,11 @@
 @agate-fullscreenpopup-bg-color: @agate-popup-bg-color;
 @agate-fullscreenpopup-bg-image: @agate-bg-image;
 
+// Header
+// ---------------------------------------
+@agate-header-subtitle-text-color: @agate-black;
```

Comment:
How about we set this to be `@agate-foreground`, and then delete the night rules entirely? That way, when we redefine night's `@agate-foreground` to be white, the header automatically picks up that color?

Reply:
Seems like a good idea!